src/StackExchange.Redis/RedisFeatures.cs

```diff
@@ -54,7 +54,7 @@ public RedisFeatures(Version version)
         /// <summary>
         /// Does BITOP / BITCOUNT exist?
         /// </summary>
-        public bool BitwiseOperations => Version >= v2_5_10;
+        public bool BitwiseOperations => Version >= v2_6_0;
```

Comment:
I looked it up, and indeed: various late 2.5.something are aka 2.6 release candidates; this simplification suffices, I think